### DISCUSSION TITLE
赞助完咋进入QQ群

### DISCUSSION THREAD

**Discussion by ZTbiubiubiu** [Meloong-Git/PCL#1939] [讨论]
如题

**Comment by iPodtouch6557** [comment]
~麻烦你把这玩意open行吗~在爱发电的bot聊天栏里发【加群】

**Reply by WForst-Breeze** [reply]
可能这位朋友已经解决了这个问题所以才 close 的……

**Reply by iPodtouch6557** [reply]
> 可能这位朋友已经解决了这个问题所以才 close 的……

我认为他可以说明啊，要不然我会知道他解没解决吗……

**Comment by ZTbiubiubiu** [comment]
问题已解决，谢谢啦，是之前我以为删掉了抱歉啊。
